pv_site_api/enode_auth.py

```diff
@@ -0,0 +1,38 @@
+import os
+
+import httpx
+
+
+def get_enode_access_token() -> str:
+    # TODO: Add environment variables
+    client_id = os.getenv("CLIENT_ID")
+    client_secret = os.getenv("CLIENT_SECRET")
+    url = os.getenv("ENODE_TOKEN_URL")
+
+    # Encode the client_id and client_secret using Basic Auth
+    auth = httpx.BasicAuth(username=client_id, password=client_secret)
```

Comment:
While this is explicit and obvious, it might be unncessary to create an instance of`httpx.BasicAuth`. It seems like the [docs say](https://www.python-httpx.org/quickstart/#authentication:~:text=To%20provide%20Basic%20authentication%20credentials%2C%20pass%20a%202%2Dtuple%20of%20plaintext%20str%20or%20bytes%20objects%20as%20the%20auth%20argument%20to%20the%20request%20functions%3A) that you can just pass in a tuple with the username and password to the `auth` parameter of `httpx.<method>`.

I'm not sure what's best here, the explicit way or to be simpler? I think simpler might be better in the long run since it's less code to read.